test_scripts/API/Navigation/SendLocation/008_SendLocation_invalid_json.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/24
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/TRS/embedded_navi/SendLocation_TRS.md
+-- Item: Use Case 1: Main Flow (Exception 1: wrong json)
+--
+-- Requirement summary:
+-- App requests SendLocation without deliveryMode and with other allowed and valid parameters
+--
+-- Description:
+-- App sends SendLocation with invalid payload (invalid json)
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered on SDL
+
+-- Steps:
+-- App sends SendLocation with invalid payload (invalid json)
+
+-- Expected:
+
+-- SDL validates parameters of the request
+-- SDL responds INVALID_DATA, success: false
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSendLocation = require('test_scripts/API/SendLocation/commonSendLocation')
+
+--[[ Local Variables ]]
+
+--[[ Local Functions ]]
+local function sendLocation(self)
+    self.mobileSession1.correlationId = self.mobileSession1.correlationId + 1
+    local msg = {
+        serviceType      = 7,
+        frameInfo        = 0,
+        rpcType          = 0,
+        rpcFunctionId    = 39,
+        rpcCorrelationId = self.mobileSession1.correlationId,    
+        --<<-- Missing :
```

Comment:
@Itileda It's better to move this comment to next line after the code

Reply:
@dboltovskyi , fixed in 6f2e7484bc3d0171c93beefc946839185e141366